src/download.php

```diff
@@ -450,6 +518,43 @@
 
 
 
+if (isset($sFilter) && $sFilter == 'gene_public') {
+    // Ugly hack to run otherwise ignored filter of individualid on screenings.
+    // FIXME: make sure this filter is part of the query to get screenings from DB.
```

Comment:
Hi @mkroon1 , can you tell me what you mean with this FIXME? I have solved the other FIXME (see a5e0ed3) but I don't understand this one.